src/Lib.hs

```diff
@@ -75,24 +75,24 @@ collectEmails = do
 
 
 processTicket :: TicketId -> App [ZendeskResponse]
```

Comment:
We need to fix this. We should take the last comment attachments and process the last attachment.